src/circuits/runcircuit.jl

```diff
@@ -106,33 +110,46 @@ function qubits(sites::Vector{<:Index}, states::Vector{String};
   return ψ
 end
 
-""" 
-    resetqubits!(M::Union{MPS,MPO})
-
-Reset qubits to the initial state:
-- `|ψ⟩=|0,0,…,0⟩` if `M = MPS`
-- `ρ = |0,0,…,0⟩⟨0,0,…,0|` if `M = MPO`
-"""
-function resetqubits!(M::Union{MPS,MPO})
-  indices = [firstind(M[j], tags = "Site", plev = 0) for j in 1:length(M)]
-  M_new = qubits(indices, mixed = !(M isa MPS))
-  M[:] = M_new
-  return M
-end
-
 """
-    circuit(sites::Vector{<:Index}) = MPO(sites, "Id")
+    trivialprocess(N::Int; mixed::Bool = false)
 
-    circuit(N::Int) = circuit(siteinds("Qubit", N))
+    trivialprocess(sites::Vector{<:Index}; mixed::Bool=false)
 
 Initialize a circuit MPO
 """
-identity_mpo(sites::Vector{<:Index}) = MPO(sites, "Id")
+trivialprocess(N::Int;  mixed::Bool = false) = 
+  trivialprocess(siteinds("Qubit", N); mixed = mixed)
 
-identity_mpo(N::Int) = identity_mpo(siteinds("Qubit", N))
+trivialprocess(M::Union{MPS, MPO,LPDO}; mixed::Bool = false) = 
+  trivialprocess(hilbertspace(M); mixed = mixed)
+
+function trivialprocess(sites::Vector{<:Index}; mixed::Bool = false) 
```

Comment:
For the `mixed = true` case, it seems like it would be easier to first implement a separate `trivialprocess(LPDO, sites)` which returns an LPDO `C` for the Choi matrix, and then call `MPO(C)`. This seems like it must be code duplication for other functionality we already have.